packages/office-ui-fabric-react/src/components/Breadcrumb/__snapshots__/Breadcrumb.test.tsx.snap

```diff
@@ -1916,6 +1919,7 @@ exports[`Breadcrumb basic rendering renders breadcumb correctly 6 1`] = `
                 className=
                     ms-Button
                     ms-Button--icon
+                    ms-Button-hasMenu
```

Comment:
as this is a variant, the format (as you see on line 1921) is 2 dashes; `ms-Button--hasMenu`